constants/errors.go

```diff
@@ -0,0 +1,7 @@
+package constants
+
+import "errors"
+
+var (
+	ErrMovieNotFoundInDB = errors.New("Movie Information for given UUID is not found in Database")
```

Comment:
Db related errors can be declared in the dao itself.
variable name can just be `ErrMovieNotFound`
Similarly, error message too can be `Movie Information for given UUID is not found in Database`

If we need to know if the movie exists in database or not, we could log such details.
Such details that are of use to us alone(developers) need not be returned as error, can just be logged.